src/utils/testUtils.tsx

```diff
@@ -37,9 +39,11 @@ const customRender: CustomRender = (
       }}
     >
       <Provider store={store}>
+        {/* <CMSApolloProvider value={createCmsApolloClient()}> */}
```

Comment:
I assume that the commented out lines are meant to be remoeved.

Reply:
yea